src/util/recorder.ts

```diff
@@ -426,15 +429,13 @@ export class Recorder {
       case "net::ERR_ABORTED":
         // check if this is a false positive -- a valid download that's already been fetched
         // the abort is just for page, but download will succeed
-        if (type === "Document" && reqresp.isValidBinary()) {
+        if (
+          type === "Document" ||
+          (type === "Media" && reqresp.isValidBinary())
+        ) {
```

Comment:
Just double-checking, did you mean to remove the check for `reqresp.isValidBinary() for downloads of type `Document` or should we shift the parentheses around a bit?